src/scss/typography/_body.scss

```diff
@@ -39,6 +39,34 @@ a {
   }
 }
 
+table:not([class]) {
```

Comment:
Please can you add a story in `typography.stories` so that users can see how the default table style will look?